packages/editor/src/client/routes/commentRatingEditView.tsx

```diff
@@ -0,0 +1,250 @@
+import {ApolloError} from '@apollo/client'
+import PlusIcon from '@rsuite/icons/legacy/Plus'
+import SpinnerIcon from '@rsuite/icons/legacy/Spinner'
+import TrashIcon from '@rsuite/icons/legacy/Trash'
+import React, {useCallback, useEffect, useState} from 'react'
+import {useTranslation} from 'react-i18next'
+import {
+  Button,
+  FlexboxGrid,
+  Form,
+  IconButton,
+  Loader,
+  Message,
+  Modal,
+  SelectPicker,
+  toaster
+} from 'rsuite'
+
+import {
+  CommentRatingSystemAnswer,
+  FullCommentRatingSystem,
+  RatingSystemType,
+  useCreateRatingSystemAnswerMutation,
+  useDeleteRatingSystemAnswerMutation,
+  useRatingSystemLazyQuery,
+  useUpdateRatingSystemMutation
+} from '../api/index'
+import {IconButtonTooltip} from '../atoms/iconButtonTooltip'
+import {createCheckedPermissionComponent} from '../atoms/permissionControl'
+
+const showErrors = (error: ApolloError): void => {
+  toaster.push(
+    <Message type="error" showIcon closable duration={3000}>
+      {error.message}
+    </Message>
+  )
+}
+
+function CommentRatingEditView() {
+  const [ratingSystem, setRatingSystem] = useState<FullCommentRatingSystem | null>(null)
+  const [answerToDelete, setAnswerToDelete] = useState<string | null>(null)
+
+  const [fetchRatingSystem, {loading: isFetching}] = useRatingSystemLazyQuery({
+    onError: showErrors,
+    onCompleted: data => setRatingSystem(data.ratingSystem)
```

Comment:
It would be nice, if the user get's a feedback in the form of a toast message.

Reply:
Nice catch!

Reply:
Solved in the update below! Thanks!